grails-app/assets/javascripts/csiBenchmark/csiBenchmarkChart.js

```diff
@@ -113,32 +118,55 @@ OpenSpeedMonitor.ChartModules.CsiBenchmarkChart = (function (chartIdentifier) {
             .style("text-anchor", "end")
             .attr("dy", "2em")
             .attr("transform", function (d) {
-                return "translate(" + barWidth / 2 + "," + (height - yScale(d.value)) + ") rotate(-45)"
+                return "translate(" + barWidth / 2 + "," + (height) + ") rotate(-45)"
```

Comment:
For uniformity you should also move this to the update section. Same applies to the other function.